.github/workflows/test.yml

```diff
@@ -42,13 +42,15 @@ jobs:
         PATH: ''
 
     - name: Setup MSBuild (x64)
+      if: always()
       id: setup_msbuild_path_x64
       uses: ./
       with:
         vs-prerelease: true
         msbuild-architecture: 'x64'
 
     - name: Setup MSBuild (arm64)
+      if: always()
```

Comment:
Since I can't comment on lines not in the PR, should you add a similar condition to `echo msbuild path` below as well? Also, can these tools only be installed on ARM64 machines? I'm not actually sure myself, and you changed the README to say the success of some of these steps depend on the agent OS so I'm expecting they are conditional. If so, can these be conditioned on the agent architecture? Not sure if such a thing is possible, but maybe.

Reply:
In this instance I was just making sure all steps executed :-) regardless of fail/succeed -- meaning I expect failure to happen on arm64 on all of these unless self-hosted (at least right now)